os/hal/ports/STM32/LLD/USARTv2/hal_serial_lld.h

```diff
@@ -437,6 +437,12 @@
 /* Driver data structures and types.                                         */
 /*===========================================================================*/
 
+typedef void (*uartirq_t)(void* sd);
+
+#define _serial_irq_code(sd) {                                      \
+  if ((sd)->config->irq_cb != NULL)                                 \
+    (sd)->config->irq_cb((sd)->config->ctx);                        \

```

Comment:
especially in situations like this it's good to include the { } under the if.

Reply:
@magicrub unfortunately that is programming style in most of ChibiOS.